helm/oncall/values.yaml

```diff
@@ -634,16 +634,26 @@ grafana:
   enabled: true
   grafana.ini:
     server:
-      domain: example.com
-      root_url: "%(protocol)s://%(domain)s/grafana"
+      domain: helm-testing-grafana
+      root_url: "%(protocol)s://%(domain)s/grafana/"
       serve_from_sub_path: true
+    feature_toggles:
+      enable: externalServiceAccounts
   persistence:
     enabled: true
   # Disable psp as PodSecurityPolicy is deprecated in v1.21+, unavailable in v1.25+
   rbac:
     pspEnabled: false
   plugins:
     - grafana-oncall-app
+  extraVolumes:
+    - name: provisioning
+      configMap:
+        name: helm-testing-grafana-plugin-provisioning
+  extraVolumeMounts:
+    - name: provisioning
+      mountPath: /etc/grafana/provisioning/plugins/grafana-oncall-app-provisioning.yaml
+      subPath: grafana-oncall-app-provisioning.yaml
```

Comment:
The plugin provisioning logic should be done in the templates. If someone overrides `extraVolumes` or `extraVolumeMounts` this won't work.